docs/guides/advanced/schemas.md

```diff
@@ -459,6 +459,112 @@ In your sync formula you only need to populate the fields of the reference objec
+
+- [`titleProperty`][titleProperty]
+- [`subtitleProperties`][subtitleProperties]
+- [`snippetProperty`][snippetProperty]
+- [`linkProperty`][linkProperty]
+- [`imageProperty`][imageProperty]
+
+To be eligible to be displayed as a card, the schemas must meet all of the following criteria:
+
+--8<-- "guides/advanced/.card_schema.md"
+
+Read the [Cards guide][cards] for more information on how these fields are used.
+
+
+### Property labels {: #labels}
+
+A card's subtitle contains a set of properties, joined with a separator. Unlike other areas of the card where the property's value is shown, in the subtitle a label is shown instead. By default this label takes the form of `{name}: {value}`.
+
+```ts
+let MovieSchema = coda.makeObjectSchema({
+  // ...
+  subtitleProperties: ["director", "year", "rating"],
+});
+```
+
+<img src="../../../images/schemas_labels_default.png" srcset="../../../images/schemas_labels_default_2x.png 2x" class="screenshot" alt="Default labels for subtitle properties">
+
+However, there are times when the default label isn't a great fit. For instance you may want to put the property name after the value (`10 bugs` instead of `Bugs: 10`) or remove the label completely (`P1` instead of `Priority: P1`).
+
+To customize the label, when specifying the `subtitleProperties` pass a [`PropertyIdentifierDetails`][PropertyIdentifierDetails] object instead of a string. Set the `property` field to the path of the property (what the string value normally contains) and set the `label` field to the template string to use for the label.
+
+There are three options for setting the label:
+
+1.  Pass a string which will be used as an alternative label.
+1.  Pass a string containing the constant [`PropertyLabelValueTemplate`][PropertyLabelValueTemplate], which acts as a placeholder for where the property's value should be inserted.
+1.  Pass an empty string, which will remove the label completely.
+
+```ts
+let MovieSchema = coda.makeObjectSchema({
+  // ...
+  subtitleProperties: [
+    "director",
+    // Fully customize the label for the year property.
+    { property: "year", label: `Released in ${coda.PropertyLabelValueTemplate}` },
+    // Only show the value of the rating property.
+    { property: "rating", label: "" },
+  ],
+});
+```
+
+<img src="../../../images/schemas_labels_custom.png" srcset="../../../images/schemas_labels_custom_2x.png 2x" class="screenshot" alt="Custom labels for subtitle properties">
+
+
+## Property paths
+
+Object schema fields that expect a property name (`titleProperty`, `snippetProperty`, etc) also accept a path to a property on a nested object. The paths are specified using a subset of the [JSONPath syntax][jsonpath], which at it's simplest just joins the property names with a dot (like `property1.property2`).
+
+!!! warning "Only supported for card fields"
+    At the moment property paths can only be used in the object [schema fields used by cards](#cards).
```

Comment:
can you leave todos in markdown? and if so can leave a todo for em to update

Reply:
Done.